node/p2p/pubsub.go

```diff
@@ -29,6 +31,7 @@ func PubSub(cfg Config) func(pubSubParams) (*pubsub.PubSub, error) {
 		opts := []pubsub.Option{
 			pubsub.WithPeerExchange(cfg.PeerExchange || cfg.Bootstrapper),
 			pubsub.WithDirectPeers(fpeers),
+			pubsub.WithMessageIdFn(hashMsgID),
```

Comment:
```suggestion
			pubsub.WithMessageIDFn(hashMsgID),
```

Reply:
This should be transferred to pubsub. The func is not defined by us